tests/step_defs/test_retrieve.py

```diff
@@ -40,7 +40,10 @@ def time_range(from_date, to_date):
 @then('the generated file must contain <entry_count> entries')
 @pytest.mark.asyncio
 def ensure_results(mocker, event_loop, output_format, time_range, entry_count):
-    """Ensure we get the right amount of entries."""
+    """Ensure we get the right amount of entries.
+       Don't compare the parsed notes to entry."""
     result, _ = event_loop.run_until_complete(apd.async_retrieve(pages=-1, **time_range))
     assert result is not None
+    if 'Notes' in result:
```

Comment:
For this test, the fields don't matter, only the number of entries. You can safely remove this.

Reply:
It will still fail if the field isn't removed, because result will be one entry longer than expected

Reply:
The test might be be incorrect then. It is supposed to test the number of entries retrieved within a period of time (a `list` of `dict`), therefore the length of an entry should not impact it.